apps/content/api/public/recitations_detail.py

```diff
@@ -1,27 +1,30 @@
+from typing import Literal
+
 from django.shortcuts import get_object_or_404
 from ninja import Schema
 from ninja.pagination import paginate
-from pydantic import Field
 
 from apps.content.models import Asset, RecitationSurahTrack, Resource
+from apps.core.mixins.constants import QURAN_SURAHS
 from apps.core.ninja_utils.request import Request
 from apps.core.ninja_utils.router import ItqanRouter
 from apps.core.ninja_utils.tags import NinjaTag
+from config.settings.base import CLOUDFLARE_R2_PUBLIC_BASE_URL
 
 router = ItqanRouter(tags=[NinjaTag.RECITATIONS])
 
 
 class RecitationSurahTrackOut(Schema):
     surah_number: int
-    surah_name_en: str
     surah_name: str
-    chapter_number: int | None = None
-    audio_url: str | None = Field(
-        None,
-        description="Absolute URL to the per-surah audio file (MP3)",
-    )
+    surah_name_en: str
+    audio_url: str
     duration_ms: int
     size_bytes: int
+    revelation_order: int
+    revelation_place: Literal["Makkah", "Madinah"]
+    ayahs_count: int
+    ayahs_timings: list[dict]
```

Comment:
`list[dict]` is vague, dict should be a specific schema, so the user would know exactly what is being returned